tests/libraries/math/test_safeMathUint256.py

```diff
@@ -4,14 +4,25 @@
 from ethereum.tools.tester import TransactionFailed
 from pytest import fixture, mark, lazy_fixture, raises
 
+@fixture(scope='session')
+def testerSnapshot(sessionFixture):
+    sessionFixture.uploadAndAddToController('solidity_test_helpers/SafeMathUint256Tester.sol')
+    return sessionFixture.chain.snapshot()
+
+@fixture
+def testerContractsFixture(sessionFixture, testerSnapshot):
+    sessionFixture.chain.revert(testerSnapshot)
```

Comment:
I believe you need to do `sessionFixture.resetSnapshot()` before this as well (though I'm less confident than above).